editoast/src/views/timetable/stdcm.rs

```diff
@@ -157,7 +160,15 @@ struct InfraIdQueryParam {
     infra: i64,
 }
 
-/// Compute a STDCM and return the simulation result
+/// This function computes a STDCM and returns the result.
+/// It first checks user authorization, then retrieves timetable, infrastructure,
+/// train schedules, and rolling stock data, and runs train simulations.
+/// The result contains the simulation output based on the train schedules
+/// and infrastructure provided.
+///
+/// If the simulation fails, the function uses a fake train to detect conflicts
```

Comment:
```suggestion
/// If the simulation fails, the function uses a direct train to detect conflicts
```

`fake` feels like it's a completely unrelated simulation. If I record correctly, it's only a simulation on the most basic pathfinding that we re-run, right?

Reply:
Done.